Makefile

```
@@ -148,7 +192,7 @@ docker-build:  ## Build the Docker image
 	docker buildx build \
 		--build-arg LIBGIT2_IMG=$(LIBGIT2_IMG) \
 		--build-arg LIBGIT2_TAG=$(LIBGIT2_TAG) \
-		--platform=$(BUILD_PLATFORMS) \
+		--platform=linux/amd64 \
```

We probably need to revert this:

```suggestion
		--platform=$(BUILD_PLATFORMS) \
```